content/docs/user-guide/contributing/docs.md

```diff
@@ -81,36 +77,60 @@ Launch the server locally with:
 $ yarn develop
 ```
 
-This will start the server on the default port, `3000`. Visit
-`http://localhost:3000/` and navigate to the docs in question. This will also
-enable the Git pre-commit hook that will be formatting your code and
+This will start the server on the default port, `8000`. Visit
+`http://localhost:8000/` and navigate to the page in question. This will also
+enable the Git pre-commit hook that will be formatting and linting your code and
 documentation files automatically.
 
-### Debugging
+### All available commands
+
+To build the project and run it:
+
+- `yarn build` - build assets in the `public` directory (`gatsby build`).
+- `yarn develop` - run development server with hot reload (`gatsby develop`).
+- `yarn start` - run production static server over the `public` directory.
+
+If you change source code files, run tests:
+
+- `yarn test` - run tests.
+
+We use [`prettier`](https://prettier.io/) to format our source code, below is a
+set of pre-built commands for your convenience. All of these checks will be
+enforced automatically upon [submitting PRs](#submitting-changes).
+
+- `yarn format-check` - check all source and content files that they are
+  properly formatted. This command does not fix any found issue, only reports
+  them.
+- `yarn format-all` - fix all found problems.
+- `yarn format-staged` - same, but only on staged files.
```

Comment:
Probably no need for users to rn this manually?
```suggestion
```

Reply:
we mention this below as well. I would say let's keep - it does not hurt and may be will help someone